docs/admin/platforms/microsoft_teams.md

```diff
@@ -5,52 +5,212 @@
 | `enable_ms_teams`            | **Yes**    | False   | Yes                       |
 | `microsoft_app_id`           | **Yes**    | --      | No                        |
 | `microsoft_app_password`     | **Yes**    | --      | No                        |
+| `microsoft_tenant_id`        | **Yes**    | --      | No                        |
```

Comment:
Breaking change or just outdated docs?

Reply:
Outdated docs, and it isn't completely required... but there is a feature that doesn't work if you don't provide this. This is something that you get by just creating the app in Azure.